controllers/kubemarkcluster.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	gocontext "context"
+	"strings"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type KubemarkCluster interface {
```

Comment:
this is a new interface representing external kubemark cluster for running kubemark pods